content/documentation/overview/index.adoc

```diff
@@ -46,68 +45,69 @@ image::graph-view.png[Demo Website]
 
 === Applications
 
-In the applications tab we can see all the apps running in our environment.
+The Applications menu entry shows all the applications running in our environment.
 
 [#img-kiali-app-list]
 .List of Applications
 image::app-list.png[Demo Website]
 {nbsp} +
-Kiali provide us all the information related with the application like the health.
+Kiali provides detailed information related to the application, such as its health or the list of its workloads.
+The health summary comes with detailed information of multiple indicators in a tooltip.
 {nbsp} +
 [#img-kiali-app-view]
-.Info of Application
+.Application Info
 image::app-view-info.png[Demo Website]
 {nbsp} +
-We can also see the metrics associated with an application provided by grafana
+Kiali also displays Istio metrics associated with an application.
 {nbsp} +
 [#img-kiali-metrics]
-.Metrics of Application
+.Application Metrics
 image::app-metrics.png[Demo Website]
 {nbsp} +
 
 === Workloads
 
-In the workload tab we have the list of workloads with their health, error rate and labels validations.
+The Workloads menu entry shows the list of workloads with their health, error rate and labels validations.
 {nbsp} +
 [#img-kiali-workload-list]
 .List of Workloads
 image::workload-list.png[Demo Website]
 {nbsp} +
-When we choose a workload we can see the information related to it and associated pods and services.
+By selecting a workload, the related information is displayed along with the associated pods and services.
 {nbsp} +
 [#img-kiali-workload-view]
-.List of Workloads
+.Workload Info
 image::workload-view-pods.png[Demo Website]
 {nbsp} +
 
 === Services
 
-In the service tab we go to the page detail of a service, we have a health icon to check if our service has some problem like error rate.
+The Services menu entry shows the list of services with their health and error rate.
 
-In this page we have an overview about the service : labels, type, ip , ports, endpoints, workloads, destiantion rules, virtual services...
+When selecting a single service, its details page includes service ip, ports, endpoints, workloads, destination rules, virtual services and more details.
 
-We can also visualize the inbound/outbound metrics for this service and go to his grafana dashboard.
+Inbound/outbound metrics are displayed for this service and a more detailed view is available in its Grafana dashboard.
 
 [#img-kiali-service-view]
-.Info of a Service
+.Service Info
 image::service-view.png[Demo Website]
 {nbsp} +
 
 === Istio Config
 
-In the Istio Config tab we have a list with all the istio objects that we have in our environment.
+Istio Config menu entry displays a list of all available Istio configuration objects that exist in user's environment.
```

Comment:
The Istio Config menu entry displays a list of all of the available Istio configuration objects that exist in the user's environment.